scvi/model/_destvi.py

```diff
@@ -110,6 +113,9 @@ def from_rna_model(
             trained CondSCVI model
```

Comment:
here we should make sure we apply to DestVI the dropout parameters from CondSCVI (for sure the dropout_decoder! they have to be the same!)